src/workshop.md

```diff
@@ -22,13 +22,15 @@ future development directions for the project.
 Please [register for the workshop](https://docs.google.com/forms/d/e/1FAIpQLSc31dyl4hUQi8TvQryPYZUjbUPAAdyVTqS0ccahCoIdpFMjrg/viewform?usp=header) online.
 The deadline to register is **Wednesday, August 27**.
 
-A couple weeks after registering, in-person attendees will receive a follow-up form for the registration fee
+* **In person:** A couple weeks after registering, in-person attendees will receive a follow-up form for the registration fee
```

Comment:
Rather than the "couple weeks" statement, can we just link directly to the payment page? https://commerce.cashnet.com/pdxMFEM

Reply:
@pazner see new commit below. Should we retain the swag sentence in the in-person bullet or not? I clarified "at a later date" b/c swag isn't in the registration fee form.